LinkedInSignIn/ApiDefinition.cs

```diff
@@ -25,32 +25,28 @@ namespace LinkedIn.SignIn
 	delegate void DeeplinkErrorBlock(NSError arg0, string arg1);
 
 
-
-	// typedef void (^APIErrorBlock)(LISDKAPIResponse *, NSError *);
-
 	// @interface LISDKSessionManager : NSObject
-	[BaseType(typeof(NSObject))]
-	interface LISDKSessionManager
+	[BaseType(typeof(NSObject), Name = "LISDKSessionManager")]
+	interface SessionManager
 	{
```

Comment:
I am not sure if you know about the `Name` property. The IDE doesn't, but you can use it to specify the exact Objective-C name to use, but provide a different one for .NET/Xamarin.

This is always cool if there is a nasty prefix... such as "LISDK". We don't need this prefix as we have the novel concept of "namespaces" :)